__tests__/app.test.js

```diff
@@ -282,6 +283,96 @@ describe("GET /api/users", () => {
+  test("200: Responds with an ascending array of articles by title", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title&order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("title", {
+          descending: false,
+        });
+      });
+  });
+  test("200: Responds with a descending array of articles by topic", () => {
+    return request(app)
+      .get("/api/articles?sort_by=topic&order=desc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("topic", {
+          descending: true,
+        });
+      });
+  });
+  test("200: Responds with an ascending array of articles by author", () => {
+    return request(app)
+      .get("/api/articles?sort_by=author&order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("author", {
+          descending: false,
+        });
+      });
+  });
+  test("200: Responds with a descending array of articles by created_at", () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at&order=desc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+  test("200: Responds with an ascending array of articles by article_id", () => {
+    return request(app)
+      .get("/api/articles?sort_by=author&order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("author", {
+          descending: false,
+        });
+      });
+  });
+  test("200: Responds with a descending array of articles by votes", () => {
+    return request(app)
+      .get("/api/articles?sort_by=votes&order=desc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("votes", {
+          descending: true,
+        });
+      });
+  });
+  test("200: Responds with an ascending array of articles by comment_count", () => {
+    return request(app)
+      .get("/api/articles?sort_by=comment_count&order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("comment_count", {
+          descending: false,
+          coerce: true,
+        });
+      });
+  });
+  test("400: Responds with an error message when given an invalid sort query", () => {
+    return request(app)
+      .get("/api/articles?sort_by=christmas&order=asc")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("400: Responds with an error message when given an invalid order type", () => {
+    return request(app)
+      .get("/api/articles?sort_by=topic&order=xyz")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+});
+
```

Comment:
Really extensive testing! 

I would say it's not necessary to check all the different valid values BUT it's not going to do any harm. 

I would maybe suggest changing at least one of them to just have a single query passed - it will still work but just to prove that your endpoint isn't dependant on always receiving two queries! (so just check sort_by or order)